reframe/core/schedulers/torque.py

```diff
@@ -35,6 +35,14 @@
 class TorqueJobScheduler(PbsJobScheduler):
     TASKS_OPT = '-l nodes={num_nodes}:ppn={num_cpus_per_node}'
 
+    def _get_nodelist(self, job, stdout):
+        nodelist_match = re.search(r'exec_host = (?P<nodelist>\S+)', stdout)
+        if nodelist_match:
+            nodelist = [x.split('/')[0]
+                        for x in nodelist_match.group('nodelist').split('+')]
+            nodelist.sort()
```

Comment:
@ZQyou Do you want the list sorted here? Because in the Slurm backend we don't do that.